MapleServer2/Constants/Skills/SkillTreeOrdered.cs

```diff
@@ -0,0 +1,270 @@
+        private static readonly int[] KnightDefault = new int[]
+        {
+            10100061,10100091
+        };
+        private static readonly int[] BerserkerDefault = new int[]
+        {
+            10200031,10200091
+        };
+        private static readonly int[] WizardDefault = new int[]
+        {
+            10300061,10300081
+        };
+        private static readonly int[] PriestDefault = new int[]
+        {
+            10400111,10400151
+        };
+        private static readonly int[] ArcherDefault = new int[]
+        {
+            10500111,10500151
+        };
+        private static readonly int[] HeavyGunnerDefault = new int[]
+        {
+            10600071,10600081
+        };
+        private static readonly int[] ThiefDefault = new int[]
+        {
+            10700041,10700042,10700061
+        };
+        private static readonly int[] AssassinDefault = new int[]
+        {
+            10800071,10800081
+        };
+        private static readonly int[] RunebladeDefault = new int[]
+        {
+            10900041,10900051
+        };
+        private static readonly int[] StrikerDefault = new int[]
+        {
+            11000051,11000053,11000055,11000081
+        };
+        private static readonly int[] SoulBinderDefault = new int[]
+        {
+            11100051,11100061
+        };
+        #endregion
+        /// <summary>
+        /// Get the specific ordered skill list of each Job. Requiered for SkillBookTree
+        /// </summary>
+        public static int[] GetListOrdered(Job job)
+        {
+            switch (job)
+            {
+                case Job.Knight:
+                    return Knight;
+                case Job.Berserker:
+                    return Berserker;
+                case Job.Wizard:
+                    return Wizard;
+                case Job.Priest:
+                    return Priest;
+                case Job.Archer:
+                    return Archer;
+                case Job.HeavyGunner:
+                    return HeavyGunner;
+                case Job.Thief:
+                    return Thief;
+                case Job.Assassin:
+                    return Assassin;
+                case Job.Runeblade:
+                    return Runeblade;
+                case Job.Striker:
+                    return Striker;
+                case Job.SoulBinder:
+                    return SoulBinder;
+                case Job.GameMaster:
+                    return GameMaster;
+                case Job.None:
+                    return None;
+                default:
+                    return None;
+            }
+        }
+
+        /// <summary>
+        /// Get the specific Split value for each Job. Requiered for SkillBookTree
+        /// </summary>
+        public static byte GetListOrderedSplit(Job job)
+        {
+            switch (job)
+            {
+                case Job.Knight:
+                    return (byte) JobSkillSplit.Knight;
```

Comment:
any reason why these are bytes?

Reply:
Is because on the `SkillTreePacket` is being pass as byte